lib/logs.coffee

```diff
@@ -244,6 +246,7 @@ getLogs = (deps, opts) ->
 	# @function
 	# @public
 	# @memberof resin.logs
+	# @deprecated .clear() will be removed in the next major release
```

Comment:
Ultra-NIT: I would use `this method` to be consistent with the rest deprecation warning of the SDK